src/DFTK.jl

```diff
@@ -172,6 +172,8 @@ include("postprocess/chi0.jl")
 export compute_current
 include("postprocess/current.jl")
 
+include("workarounds/forwarddiff_rules.jl")
```

Comment:
This needs to be conditionally included when ForwardDiff is available (using requires, I can do it if you want).